user/views.py

```diff
@@ -1 +1,23 @@
-# write your code here
+from rest_framework import generics
+from .serializers import UserSerializer
```

Comment:
import are not ordered correctly.
Imports should be grouped in the following order:

1. Standard library imports.
2. Related third party imports.
3. Local application/library specific imports.